### ISSUE TITLE
Method to include external text

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/isodoc-models#17]
Such as the "include" syntax of AsciiDoc? e.g., when "SOURCEing" definitions we directly pull in the text?

**Comment by opoudjis** [comment]
In XML? That would be an xi:include. I'd rather the AsciiDoc be what's doing the pulling, and the XML being self-contained. Pulling the definition is going to be a bear in XML, and at least AsciiDoc has a tag syntax to do it with.

**Comment by ronaldtse** [comment]
That's true. In an imaginary format we should allow importing of such, like:

`iso27000.terms["confidentiality"].definition`
